tree/ntuple/v7/doc/specifications.md

```diff
@@ -253,11 +253,9 @@ The type can take one of the following values
 | Type | Meaning       | Payload format                      |
 |------|---------------|-------------------------------------|
 | 0x01 | Large locator | 64bit size followed by 64bit offset |
-| 0x02 | DAOS locator  | Object64                            |
 
 Each locator type follows a given format for the payload (see Section "Well-known payload formats" below).
-The range 0x03 - 0x7f is currently unused.
-Additional types can be registered in the future.
+The range 0x03 - 0x7f is reserved for future use.
```

Comment:
Why are we not reserving the range from 0x02? Is that to avoid breaking existing code? Are we going to reserve it in epoch 1?

Reply:
Ah, sorry! Typo.